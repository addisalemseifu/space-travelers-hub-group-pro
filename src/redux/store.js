import { configureStore } from '@reduxjs/toolkit';
import logger from 'redux-logger';
import missions from './missions/missionSlice';
import dragonsReducer from './dragons/dragonSlice';

const store = configureStore({
  reducer: {
    missions,
    dragons: dragonsReducer,
  },
  middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger),

});

export default store;
