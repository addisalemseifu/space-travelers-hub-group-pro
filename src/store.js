import { configureStore } from '@reduxjs/toolkit';
import dragonsReducer from './redux/dragons/dragonSlice';

const store = configureStore({
  reducer: {
    dragons: dragonsReducer,
  },
});

export default store;
