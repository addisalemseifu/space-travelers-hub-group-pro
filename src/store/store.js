import { configureStore } from '@reduxjs/toolkit';
import dragonsReducer from './dragon/dragonSlice';

const store = configureStore({
  reducer: {
    dragons: dragonsReducer,
  },
});

export default store;
