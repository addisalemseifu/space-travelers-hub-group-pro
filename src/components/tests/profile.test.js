import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Profile from '../../components/profile';
import store from '../../redux/store';

it('renders correctly', () => {
  const profile = renderer
    .create(
      <Provider store={store}><Profile /></Provider>,
    )
    .toJSON();
  expect(profile).toMatchSnapshot();
});