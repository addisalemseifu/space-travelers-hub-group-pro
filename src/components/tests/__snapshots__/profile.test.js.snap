// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="pro-container"
>
  <div>
    <h3>
      My Rockets
    </h3>
  </div>
  <div
    className="dragons-list"
  >
    <h3>
      My Dragons
    </h3>
    <table>
      <tbody>
        <p>
          You have no reserved dragons
        </p>
      </tbody>
    </table>
  </div>
  <div
    className="reserve-wrapper"
  >
    <div
      className="missions-profile"
    >
      <h3>
        My Missions
      </h3>
      <p>
        You have no reserved missions
      </p>
    </div>
  </div>
</div>
`;
